src/Bot/commands/commandsReplies.ts

```diff
@@ -16,7 +16,7 @@ const graphQLClient = new GraphQLClient(config.graphqlAPI);
 
 export const lookupReply = async (interaction: CommandInteraction) => {
   try {
-    const usernameArg = interaction.options.getString("username");
+    const usernameArg = interaction.options.get("username");
```

Comment:
`getString` is getting omitted for `CommandInteractionOptionResolver` which only has `get`, `getUser`, and `getMember`:

```ts
  public options: Omit<
    CommandInteractionOptionResolver<Cached>,
    | 'getMessage'
    | 'getFocused'
    | 'getMentionable'
    | 'getRole'
    | 'getAttachment'
    | 'getNumber'
    | 'getInteger'
    | 'getString'
    | 'getChannel'
    | 'getBoolean'
    | 'getSubcommandGroup'
    | 'getSubcommand'
  >;
```